src/flow/ext_multiroute_flow.jl

```diff
@@ -181,7 +181,7 @@ function slope{T<:AbstractFloat}(
   )
   slope = 0
   for e in edges(flow_graph)
-    if cut[dst(e)] - cut[src(e)] > 0 &&
+    if cut[dst(e)] == 2 > cut[src(e)] &&
```

Comment:
The Boykov-Kolmogorov max-flow algorithm also provides a min-cut solution. Nodes that are on the side of the source node has a value of 1, the one on the side of the target a value of 2, and the one can be any side a value of 0.
Here I check how many edges are going from one side of the cut to another and with a capacity value bigger than some real `restriction`. More precisely from the side of the source to the side of the target.